content/docs/command-reference/exp/run.md

```diff
@@ -195,29 +207,85 @@ experiment we just ran (`exp-44136`).
 ## Example: Modify parameters on-the-fly
 
 `dvc exp run --set-param` (`-S`) saves you the need to manually edit the params
-file before running an experiment. It can override (`train.epochs=10`), append
-(`+train.weight_decay=0.01`), or remove (`~model.dropout`) parameters.
+file before running an experiment.
 
-<admon type="tip">
+It can override (`train.epochs=10`), append (`+train.weight_decay=0.01`), or
+remove (`~model.dropout`) parameters.
 
-DVC supports the
-[Hydra Override Grammar](https://hydra.cc/docs/advanced/override_grammar/basic/),
-with the exception of patterns that modify Hydra's defaults list.
+You can modify multiple parameters at the same time:
 
-</admon>
+```dvc
+dvc exp run -S 'prepare.split=0.1' -S 'featurize.max_features=100'
+...
+```
+
+## Example: Grid Search
+
+Combining `--set-param` and `--queue`, we can perform a
+[Grid search](https://en.wikipedia.org/wiki/Hyperparameter_optimization#Grid_search)
+for tuning hyperparameters.
 
-You can optionally provide a prefix `[<filename>:]` in order to edit a specific
-`dvc params` file. If not provided, `params.yaml` will be used as default.
+DVC supports Hydra's
+[Choice](https://hydra.cc/docs/advanced/override_grammar/extended/#choice-sweep)
+and
+[Range](https://hydra.cc/docs/advanced/override_grammar/extended/#range-sweep)
+syntax for adding multiple experiments to the queue.
+
+This syntax can be used for multiple parameters at the same time, adding all
+combinations to the queue:
 
 ```dvc
-$ dvc exp run -S train_config.json:+train.weight_decay=0.001
+$ dvc exp run -S 'train.min_split=2,8,64' -S 'train.n_est=100,200' --queue
+Queueing with overrides '{'params.yaml': ['train.min_split=2', 'train.n_est=100']}'.
+Queued experiment 'ed3b4ef' for future execution.
+Queueing with overrides '{'params.yaml': ['train.min_split=8', 'train.n_est=100']}'.
+Queued experiment '7a10d54' for future execution.
+Queueing with overrides '{'params.yaml': ['train.min_split=64', 'train.n_est=100']}'.
+Queued experiment '0b443d8' for future execution.
+Queueing with overrides '{'params.yaml': ['train.min_split=2', 'train.n_est=200']}'.
+Queued experiment '0a5f20e' for future execution.
+Queueing with overrides '{'params.yaml': ['train.min_split=8', 'train.n_est=200']}'.
+Queued experiment '0a5f20e' for future execution.
+Queueing with overrides '{'params.yaml': ['train.min_split=64', 'train.n_est=200']}'.
+Queued experiment '0a5f20e' for future execution.
+$ dvc queue start
 ...
 ```
 
+We can then find and apply the best experiment:
+
 ```dvc
+$ dvc exp apply $(dvc exp show --no-pager --sort-by avg_prec | tail -n 2 | head -n 1 | grep -o 'exp-\w*')
+```
+
+<admon type="tip">
+
+See more in `dvc exp apply` and `dvc exp show`
+
+</admon>
+
+## Example: Modify parameters of custom files
```

Comment:
```suggestion
## Example: Append parameters from custom files
```

Seems like this example is both showing custom files and the `+` syntax. I understand you may not want a separate example just to introduce `+` since it's strictly about teaching Hydra overrides, but let's at least be clear that this example adds a new parameter.